rpm/pacemaker.spec.in

```diff
@@ -636,9 +636,6 @@ find %{buildroot} -name '*.la' -type f -print0 | xargs -0 rm -f
 # Byte-compile Python sources where suitable and the distro procedures known
 %if %{defined py_byte_compile}
 %{py_byte_compile %{python_path} %{buildroot}%{_datadir}/pacemaker/tests}
```

Comment:
Is this line still needed? I don't think anything installed here is importable

Reply:
No, doesn't look like it.  I'll update.